etna/whatson/views.py

```diff
@@ -0,0 +1,18 @@
+from django.http import JsonResponse
+from django.views.decorators.csrf import csrf_exempt
+
+
```

Comment:
Too much whitespace?

Reply:
Whitespace is enforced by our formatters - they leave a double line after our imports 😃